aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
```

Comment:
mmm ... I think a "reverse-proxy" just describes how the IIS server is behaving in the setup, while "middleware" is the actual code bits that make it work. IIS is a "reverse-proxy server" in the config of the network IIRC.

These concepts sort of blend into each other, so it's a kind'a chicken-and-egg thing, too.